### PR TITLE
Fixes #15604: Fix file manager navbar's display

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://issues.rudder.io/issues/15604

### CODE REVIEW

File: builder/index.html [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
     <head>
         <link type="image/x-icon" rel="icon" href="ncf-builder/css/favicon.ico"/>
-
         <link href="css/angular-filemanager-1.5.1-custom.css?version=dev" rel="stylesheet">
+        <link href="css/glyphicons/css/glyphicons.min.css?version=dev" rel="stylesheet">
         <link href="css/bootstrap-4.0.0.min.css?version=dev" rel="stylesheet">
         <link rel="stylesheet" href="css/font-awesome-5.6.1.min.css?version=dev">
         <link rel="stylesheet" href="webfonts/ionicons-2.0.1/css/ionicons.min.css?version=dev">
         <link href="css/ngToast.min.css?version=dev" rel="stylesheet">
+        <link href="css/angular-filemanager-1.5.1.min.css?version=dev" rel="stylesheet">
```
Comment (VinceMacBuche) [inline builder/index.html]: Why do you include it ? it's already done on line 5 !!

Comment (VinceMacBuche) [reply builder/index.html]: If not needed, you can also remove the new included file!!

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**